wp-content/civi-extensions/goonjcustom/Civi/MaterialContributionService.php

```diff
@@ -119,6 +98,63 @@ public static function attachContributionReceiptToEmail(&$params, $context = NUL
     $params['cc'] = 'crm@goonj.org';
   }
 
+  private static function getCityFromGoonjOfficeId($goonjOfficeId) {
```

Comment:
We don't need this function. There should be just one function `getContributionCity`,  move all the logic related to finding the city in that function

Reply:
sure